src/cms/templates/pages/page_tree_node.html

```diff
@@ -33,6 +33,13 @@
             {{ page_translation.get_status_display }}
         </a>
     </td>
+    <td>
+        <a href="{% url 'edit_page' page_id=page.id region_slug=region.slug language_slug=LANGUAGE_CODE %}" class="block py-3 px-2 text-gray-800">
```

Comment:
```suggestion
        <a href="{% url 'edit_page' page_id=page.id region_slug=region.slug language_slug=language.slug %}" class="block py-3 px-2 text-gray-800">
```
Same argument like before, `LANGUAGE_CODE` is the language code of the UI language, but we want to open the edit form in the content language here.